uber/site_sections/registration.py

```diff
@@ -274,6 +274,36 @@ def recent(self, session):
     def merch(self, message=''):
         return {'message': message}
 
```

Comment:
I think this is all straightforward, but I'll be happy to answer any questions if anything is unclear.